src/components/organisms/plans/plans.html

```diff
@@ -0,0 +1,52 @@
+<div class="container">
```

Comment:
すべてのパーツの上位階層を`container`で始めてしまうと、「ページを完成させる」で連結させた時に、各パーツで微妙に調整した`container`のスタイルが帳消しになってしまいます。
```suggestion
<div class="plans">
    <div class="container">
```
とするか
```suggestion
<div class="plans container">
```
として
`.plans .container{

 }`
とcssを書くかしてみてください。
他のパーツで指摘し忘れている箇所があれば、同様に修正お願いします。